src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.html

```diff
@@ -0,0 +1,32 @@
+<igx-tabs>
+    <igx-tab-item>
+        <igx-tab-header>
+            <span igxPrefix>Tab</span>
```

Comment:
I'd rather have the IG logo as a prefix and a close button as a suffix.

Reply:
Changed